Pod/Classes/MRGPagerTitleStrip.m

```diff
@@ -192,9 +196,16 @@ - (void)updateButtonsAnimated:(BOOL)animated {
     [self.buttons makeObjectsPerformSelector:@selector(removeFromSuperview)];
     [self.buttons removeAllObjects];
     
-    for (NSString *title in self.pageTitles) {
+    for (NSUInteger idx = 0; idx < self.pageTitles.count; idx++) {
+        NSString *title = self.pageTitles[idx];
         UIButton *button = [UIButton buttonWithType:UIButtonTypeCustom];
```

Comment:
Ideally we would do a second loop for the badges instead of recreating the buttons when they change.

Reply:
Should probably be done in `updateView`